storybookjs/storybook
# Bump react-syntax-highlighter to 13.2.1

Issue: #11890

there was new security alert for prismjs, which react-syntax-highlighter was using.

## What I did

bumped react-syntax-highlighter to 13.2.1 to address security alert for prismjs.

it's a change in major version, but the only breaking change is `dracula` theme being changed to `darcula`

https://github.com/react-syntax-highlighter/react-syntax-highlighter/releases

## How to test

- Is this testable with Jest or Chromatic screenshots?
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

n/a
